odoo/addons/test_assetsbundle/tests/test_js_transpiler.py

```diff
@@ -247,15 +247,16 @@ def test_06_import(self):
 
 import Line9  from "test.Dialog";
 import  { Line10, Notification }  from 'test.Dialog2';
+import Line11  from "test/Dialog";
 
-import * as Line11 from "test.Dialog";
+import * as Line12 from "test.Dialog";
```

Comment:
Probably should add a test instance with longer names (multiple dots) as it was a failing case as well.